fastlane.gemspec

```diff
@@ -75,7 +75,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('babosa', '>= 1.0.3', '< 2.0.0') # library for creating human-friendly identifiers, aka "slugs"
   spec.add_dependency('bundler', '>= 1.12.0', '< 3.0.0') # Used for fastlane plugins
   spec.add_dependency('CFPropertyList', '>= 2.3', '< 4.0.0') # Needed to be able to read binary plist format
-  spec.add_dependency('colored') # colored terminal output
+  spec.add_dependency('colored', '~> 1.2') # colored terminal output
```

Comment:
this version was released in 2010.  So pretty safe even though in theory bumping could still be a problem?

Reply:
I think this is fine